build/nuke/Build.Steps.cs

```diff
@@ -214,6 +214,19 @@ partial class Build
                 .EnableNoRestore()
                 .SetFramework(TargetFramework.NETCOREAPP3_1)
                 .SetOutput(TracerHomeDirectory / TargetFramework.NETCOREAPP3_1));
+
+            var source = RootDirectory / "OpenTelemetry.AutoInstrumentation.nuspec";
+            var dest = OutputDirectory / "nuget";
+
+            CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
+
+            DotNetPack(s => s
+                .SetProject(Solution.GetProject(Projects.AutoInstrumentation))
+                .SetConfiguration(BuildConfiguration)
+                .SetProperty("NuspecFile", dest / "OpenTelemetry.AutoInstrumentation.nuspec")
+                .EnableNoBuild()
+                .EnableNoRestore()
+                .SetOutputDirectory(dest));
```

Comment:
I think it should be a new build step that should be executed after `PublishNativeProfiler`, `PublishManagedProfiler`, ``CopyIntegrationsJson`. Otherwise, it can be created without native binaries and `integrations.json`.